Add the Select component to the action component list (#858)
zeit/docs: pages/docs/integrations/v2/integrations-docs-mdx/component-catalog/action-components.mdx

```jsx
<Link action="doSomething">Do Something</Link>
```

### Select

> The Select component provides the user with a drop down menu that contains the specified options.

Reviewer: ```suggestion
The Select component provides the user with a dropdown menu that contains specified options.
```

> Usually, this is an [input component](/docs/integrations#component-catalog/input-components).

Reviewer: ```suggestion
By default, this is an [input component](/docs/integrations#component-catalog/input-components),
```

> But simply adding the `action` attribute, it can be converted into an action component.

Reviewer: ```suggestion
however, adding the `action` attribute, the component can be converted into an [action](#component-catalog/action-components) component.
```

> After that, when you select an option, UIHook will be called again with the defined action.

Reviewer: ```suggestion
When using the `action` attribute, when a user selects an option from the element, the UIHook will be called again containing the context of the defined action.
```

Author: @timothyis this is not the context. It's client state.

Reviewer: ```suggestion
When using the `action` attribute, when a user selects an option from the element, the UIHook will be called again containing the defined action within the [client state](#creating-user-interfaces/client-state).
```
